src/components/TagPicker/tagPickerPropTypes.js

```diff
@@ -23,6 +24,9 @@ const propTypes = {
 
     /** Should show the selected option that is disabled? */
     shouldShowDisabledAndSelectedOption: PropTypes.bool,
+
+    /** Safe area insets required for mobile devices margins */
```

Comment:
I think we can modify this comment to explain why and how we should use this required prop. 
I would have suggested something but I don't understand this prop 😅

Reply:
It is simple insets from safe area. It is a [common RN feature](https://reactnative.dev/docs/safeareaview). Anyway, I will try to describe comments.

Reply:
Done - 19ff8d8.